olive/common/ort_inference.py

```diff
@@ -174,7 +172,7 @@ def get_ort_inference_session(
     for idx, provider in enumerate(providers):
         if provider in ["CUDAExecutionProvider", "DmlExecutionProvider"] and device_id is not None:
             provider_options[idx]["device_id"] = str(device_id)
-        elif provider == "QNNExecutionProvider":
+        elif provider == "QNNExecutionProvider" and "backend_path" not in provider_options[idx]:
```

Comment:
it will not work, getting the same error. I think it is because it is set after this. So perhaps do not set it when it is win-ort

Reply:
backend_path is set up inside ort.InferenceSession

Reply:
Then why is this even a problem. InferenceSession can just override whatever is provided from here.